client/src/components/MadeWithLove/index.ts

```diff
@@ -0,0 +1 @@
+export { MadeWithLove as MadeWithLove } from './MadeWithLove';
```

Comment:
Why did you use alias the same name as original?

Reply:
I tried multiple things here forgot to remove